build/CoreGraphics/dll/CoreGraphics.def

```diff
@@ -236,6 +236,7 @@ LIBRARY CoreGraphics
         _CGContextSetShadowProjectionTransform
         _CGContextCreateWithD2DRenderTarget
         _CGGetD2DFactory
+        _CGGetPixelFormatProperties
```

Comment:
hmmm I don't think so.

---
In reply to: [87862945](https://github.com/Microsoft/WinObjC/pull/1371#discussion_r87862945) [](ancestors = 87862945)